### ISSUE TITLE
Xiaomi Door and Window Sensor 2 Support BLE Request

### ISSUE THREAD

**Issue by DoeWayne** [AlCalzone/ioBroker.ble#459]
Would it be possible to add support for the Xiaomi Door and Window Sensor 2 devices ? 

MCCGQ02HL

**Comment by AlCalzone** [comment]
Do you get any states when the device is in range? If so, please show me the value.

**Comment by DoeWayne** [comment]
The problem is there are so many BLE devices, I do not know exactly which message depends to which device. It might be good to add for each message the device name to the messages, if possible.

This is the device address: e4:aa:ec:35:11:8e

Here are some example entries from the log, possibly not all are from that device.

ble.0	2020-10-07 22:57:00.549	debug	(6845) setting state ble.0.e4:aa:ec:35:11:8e.services.fe95
ble.0	2020-10-07 22:57:00.548	debug	(6845) e4:aa:ec:35:11:8e > got values: {"services.fe95":"30588b090e8e1135ecaae408"}
ble.0	2020-10-07 22:57:00.546	debug	(6845) _default: e4:aa:ec:35:11:8e > got data 30588b090e8e1135ecaae408 for fe95
ble.0	2020-10-07 22:57:00.544	silly	(6845) States user redis pmessage ble.0.*/ble.0.e4:aa:ec:35:11:8e.rssi:{"val":-55,"ack":true,"ts":1602104220539,"q":0,"from":"system.adapter.ble.0","user":"system.user.admin","lc":1602104220539}
ble.0	2020-10-07 22:57:00.530	debug	(6845) updating rssi state for e4:aa:ec:35:11:8e
ble.0	2020-10-07 22:57:00.525	debug	(6845) plugin _default is handling e4:aa:ec:35:11:8e
ble.0	2020-10-07 22:57:00.524	debug	(6845) has manufacturerData: false
ble.0	2020-10-07 22:57:00.524	debug	(6845) serviceData = [{"uuid":"fe95","data":{"type":"Buffer","data":[48,88,139,9,14,142,17,53,236,170,228,8]}}]
ble.0	2020-10-07 22:57:00.524	debug	(6845) has serviceData: true
ble.0	2020-10-07 22:57:00.523	debug	(6845) has advertisement: true
ble.0	2020-10-07 22:57:00.522	debug	(6845) discovered peripheral e4:aa:ec:35:11:8e

If you need more, please let me know what I can provide.

**Comment by AlCalzone** [comment]
You should be able to see that in the object browser:
![grafik](https://user-images.githubusercontent.com/17641229/95419365-c8b4a580-0939-11eb-9db0-7d91f2d17ea3.png)

That being said, the device in your log does not advertise any useful data. It might require an active connection to read the status. If you can find anything that describes how to interact with that device, it would be helpful.

**Comment by DoeWayne** [comment]
![image](https://user-images.githubusercontent.com/32009137/95514938-1b837100-09bd-11eb-9159-a877b6a0ae1b.png)
Just that.

**Comment by alexkid78** [comment]
Hallo AlCalzone,

Ich habe hier auch 4 Sensoren von Typ MCCGQ02HL und bekomme folgende Objekte.

![image](https://user-images.githubusercontent.com/76533842/103027543-5aee4200-4556-11eb-9d0d-73d9543a6d66.png)

**Comment by AlCalzone** [comment]
Ja, da steht auch nur die MAC-Adresse drin, sonst nix.

**Comment by alexkid78** [comment]
Ja, mehr kommt leider nicht. Kann ich da selbst noch etwas tuen?
- hier mal noch der 2. Sensor (gleiche Ausgabe)

![image](https://user-images.githubusercontent.com/76533842/103028178-7ad23580-4557-11eb-93f4-b3376c6e6d77.png)

VG Alex

**Comment by AlCalzone** [comment]
Wie oben schon geschrieben - es kann sein, dss eine aktive Verbindung benötigt wird. Dann ist außerdem eine Information darüber nötig, wie man an die Daten kommt. Wenn du sowas findest, könnte es helfen.

**Comment by k4sP3rsky** [comment]
@AlCalzone 
Hallo, könnte dies dir weiterhelfen?
https://github.com/custom-components/ble_monitor
![door sensor 2](https://user-images.githubusercontent.com/46832751/107852072-d067e780-6e0e-11eb-9a94-1977f2aaea0e.JPG)

**Comment by MoYzOne1** [comment]
Ich grab das mal aus. Man braucht definitiv eine aktive Verbindung. Der ioBroker Datenpunkt wird nur aktualisiert, wenn der Sensor in der Xiaomi App auf dem Handy verbunden und diese aktiv ist. Schließt man die App und damit die Verbindung, wird nichts mehr aktualisiert.
